src/js/pyodide.ts

```diff
@@ -46,6 +46,7 @@ export type ConfigType = {
   stdout?: (msg: string) => void;
   stderr?: (msg: string) => void;
   jsglobals?: object;
+  sysExecutable?: string;
```

Comment:
How about making it private? For now, I don't think it will be used in any places except for the venv.

Reply:
Well we already have public `args`, this is allowing to specify `argsv[0]`. I don't think many people will need it but it's also likely to be very stable. On the other hand there's also not much harm in making it private.